src/Uno.UWP/ApplicationModel/DataTransfer/Clipboard.Android.cs

```diff
@@ -11,10 +11,120 @@ public static partial class Clipboard
 	{
 		public static void SetContent(DataPackage content)
 		{
-			if (ContextHelper.Current.GetSystemService(Context.ClipboardService) is ClipboardManager manager)
-			{
-				manager.Text = content?.Text;
+			if(content is null)
+			{
+				throw new ArgumentNullException("cannot SetContent - null parameter");
 			}
+
+			ClipData clipData=null;
+
+			if (content.Text != null)
+			{
+				clipData = ClipData.NewPlainText("clipdata", content.Text);
+			}
+
+			if (content._uri != null)
+			{
+				clipData = ClipData.NewRawUri("clipdata", Android.Net.Uri.Parse(content._uri.ToString()));
+			}
+
+			if (content._html != null)
+			{
+				string plainText = "";
+				if(content.Text != null)
+				{
+					plainText = content.Text;
+				}
+				else
+				{
+					// simple conversion from HTML to plaintext
+					plainText = content._html;
+					for(int tagStart = plainText.IndexOf("<"); tagStart > -1; tagStart = plainText.IndexOf("<", tagStart))
+					{
+						int tagEnd = plainText.IndexOf(">", tagStart);
+						if(tagEnd <0)
+						{
+							break;	// end of loop - we have start, but without end
+						}
+
+						plainText = plainText.Remove(tagStart, tagEnd - tagStart + 1);
+					}
+				}

```

Comment:
Make this a helper method, and use a regex instead, don't invent your own loop here.

Reply:
To learn about how to use regex in UWP is too much overhead for me. It so different than regex in vi editor...

Reply:
You can most probably find one very easily on StackOverflow, even if it does not handle all cases for now.